test_utils/test_settings.py

```diff
@@ -30,28 +30,14 @@
 
 SECRET_KEY = "not-so-secret-key"
 EVENT_BUS_PRODUCER_CONFIG = {
-    'org.openedx.content_authoring.xblock.published.v1': (
+    'org.openedx.content_authoring.xblock.published.v1':
         {
-            'topic': 'content-authoring-xblock-lifecycle',
-            'event_key_field': 'xblock_info.usage_key',
-            'enabled': True
+            'content-authoring-xblock-lifecycle': {'event_key_field': 'xblock_info.usage_key', 'enabled': True},
+            'content-authoring-all-status': {'event_key_field': 'xblock_info.usage_key', 'enabled': True},
+            'content-authoring-xblock-published': {'event_key_field': 'xblock_info.usage_key', 'enabled': False}
```

Comment:
Nit: Having one item disabled in the top-level settings feels very single test specific. Other alternatives:
1. Rename the topic to something like `content-authoring-xblock-published-disabled-topic`, or some such, to make it clear that any test that needs a disabled topic could use this, or
2. Simply use` @override_settings` to set `EVENT_BUS_PRODUCER_CONFIG` the way you need it for some specific test.